Main.java

```diff
@@ -9,7 +11,18 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Creates a scanner for input
+    Scanner input = new Scanner(System.in);
+    //asks for temperature in celcius
+    System.out.println("Please enter a temperature you would like to convert in degrees Celsius.");
+    int C = input.nextInt();
+    //calcules temperature
+    int F = (C*9)/5 +32;
```

Comment:
Great work. Comments, math, and output are all fantastic. The only issue would be with the C and F variables. These are just poor names for variables because they should always begin with a lowercase letter and also they don't really describe what they are storing. C and F are really the units. Instead, call them something like degreesInC or tempInC. Otherwise, excellent work!